build.rs

```diff
@@ -0,0 +1,6 @@
+use std::process::Command;
+fn main() {
+	let output = Command::new("git").args(["rev-parse", "HEAD"]).output().unwrap();
```

Comment:
For those building from release tarballs, git information is not available. Should either fallback to a fixed value like `dev` gracefully, or allow specifying a version via an environment variable.

Reply:
Thank you for the review!